### PR TITLE
Update organizing-introduction.md

### PR DESCRIPTION
**Pull request by alvarorf**
Made some styling changes to make it more readable.

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

...your text here

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

File: javascript/organizing-js/organizing-introduction.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
 
-In the beginning, this is a great thing! For example, if you just want to make a simple button on your webpage do something you can set that up in a couple lines of code. However, as your program becomes more complex, it can become hard to maintain unless you take care to organize your code and because JavaScript is such a flexible language how you do that is entirely up to you. For many coders making decisions about design patterns is crippling so we're here to help.
+In the beginning, this is a great thing! For example, if you just want to make a simple button on your webpage do something, you can set that up in a couple of lines of code. However, as your program becomes more complex, it can become hard to maintain unless you take care of organizing your code and, because JavaScript is such a flexible language, how you do that is entirely up to you. For many coders making decisions about design patterns, this is crippling; so, we're here to help.
```
Comment (twalton83) [inline javascript/organizing-js/organizing-introduction.md:5]: >  For many coders making decisions about design patterns, this is crippling; so, we're here to help.

I think this makes the sentence very choppy.

> "For many coders, making decisions about design patterns is crippling, so we're here to help.

**Review by twalton83 — CHANGES_REQUESTED** [review]

File: javascript/organizing-js/organizing-introduction.md (hunk 1 above)
Comment (alvarorf) [reply javascript/organizing-js/organizing-introduction.md]: Thanks, I already modified the sentence according to your suggestion.

File: javascript/organizing-js/organizing-introduction.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
-In the beginning, this is a great thing! For example, if you just want to make a simple button on your webpage do something you can set that up in a couple lines of code. However, as your program becomes more complex, it can become hard to maintain unless you take care to organize your code and because JavaScript is such a flexible language how you do that is entirely up to you. For many coders making decisions about design patterns is crippling so we're here to help.
+In the beginning, this is a great thing! For example, if you just want to make a simple button on your webpage do something, you can set that up in a couple of lines of code. However, as your program becomes more complex, it can become hard to maintain unless you take care to organize your code and, because JavaScript is such a flexible language, how you do that is entirely up to you. For many coders making decisions about design patterns is crippling so we're here to help.
```
Comment (twalton83) [inline javascript/organizing-js/organizing-introduction.md:5]: This doesn't quite look like my suggestion, it just looks like you rolled the sentence back.

**Comment by CatQueenCodes** [comment]
@alvarorf This has been open for about a month and still has some requested changes that need to be addressed! Let us know if you are interested in finishing it up or not.

File: javascript/organizing-js/organizing-introduction.md (hunk 2 above)
Comment (CatQueenCodes) [reply javascript/organizing-js/organizing-introduction.md]: I went ahead and made this edit, trying to knock out all the PRs

**Review by CatQueenCodes — APPROVED** [review]

**Comment by CatQueenCodes** [comment]
This got abandoned and doesn’t seem important

**Pull request closed without merging**